keyboards/handwired/dactyl_manuform/5x8/keymaps/default/rules.mk

```
@@ -1 +1,5 @@
 DYNAMIC_MACRO_ENABLE = yes
+SPLIT_KEYBOARD = yes
+OLED_ENABLE = yes
+OLED_DRIVER = ssd1306
+OLED_TRANSPORT = i2c
```

only OLED_ENABLE is actually needed here. The rest are the same as the defaults. 
```suggestion
OLED_ENABLE = yes
```